service/src/test/java/bio/terra/pipelines/controller/PipelineRunsApiControllerTest.java

```diff
@@ -7,9 +7,7 @@
 import static org.junit.jupiter.api.Assertions.assertNull;
```

Comment:
at this point, you probably want to start reviewing with whitespace changes disabled. there were some nesting changes which i'll explain below...

https://github.com/DataBiosphere/terra-scientific-pipelines-service/pull/251/files?w=1